helpers/index.js

```diff
@@ -451,7 +445,7 @@ async function displayOneOrMultiplePagesNew(interaction, fetchFn, searchParams,
                     if (offset > 0) {
                         searchParams.set('offset', Math.max(offset - NUM_ROWS, 0));
                     } else {
-                        searchParams.set('offset', Math.floor(count / NUM_ROWS) * NUM_ROWS);
+                        searchParams.set('offset', Math.floor((count - 1) / NUM_ROWS) * NUM_ROWS);
```

Comment:
Fixed a bug where if the `count` was evenly divisible by `NUM_ROWS`, then the offset would be one factor of `NUM_ROWS` too high.